src/app/pages/OnDemandTvPage/OnDemandTvPage.tsx

```diff
@@ -174,18 +169,10 @@ const OnDemandTvPage = ({
             {`${brandTitle}, ${formattedTimestamp}`}
           </VisuallyHiddenText>
           {mediaIsAvailable ? (
-            <AVPlayer
-              embedUrl={embedUrl}
-              assetId={episodeId}
-              placeholderSrc={getPlaceholderImageUrl(imageUrl)}
-              type="video"
-              title="On-demand TV"
-              iframeTitle={iframeTitle}
-              hasBottomPadding={false}
-              skin="classic"
-              showLoadingImage
-              darkPlaceholder
+            <MediaLoader
+              blocks={pageData?.mediaBlocks}
               css={styles.mediaPlayer}
+              pageIdentifierOverride={episodePageIdentifier}
```

Comment:
Can we add a test to the OnDemandTV test suite, to ensure that when we render the OD TV page, then the MediaLoader is called with the correct pageIdentifierOverride? We do have a similar test in the Media Loader, but I would like a test at this level too, just to make sure.

Reply:
Yes we can. _No. I'm not Obama_ 😂 
We now have a test to check this [here](https://github.com/bbc/simorgh/pull/11928/files#diff-3392579f3c5412875131f2725913fc36918dade93cb4affa1cc094570a63f7a3R224).